lib/mini_stitch/api.rb

```diff
@@ -6,11 +6,11 @@ class Api
     def initialize(table_name, sequence, key_names, data)
       validate(key_names, data, table_name)
 
-      @request_params = { Authorization: "Bearer #{MiniStitch.configuration.token}" }.merge(DEFAULT_REQUEST_PARAMS)
       @table_name = table_name
       @sequence = sequence.to_i
       @key_names = key_names.map(&:to_s)
-      @data = build_records(data)
+      @data = build_records(data, MiniStitch.configuration.client_id)
+      @secondary_data = build_records(data, MiniStitch.configuration.secondary_client_id)
```

Comment:
Maybe this step can be skipped `if !MiniStitch.configuration.has_second_configuration?` ?